example/moduleE.ts

```diff
@@ -0,0 +1 @@
+console.log(123);
```

Comment:
what is this?

Reply:
i wanted to test that we handled the "push to a branch" github action flow properly, so I,
- added a js file
- created a PR to migrate it
- merged it into this branch

(That replicates what happens when we merge PRs into master.)